### ISSUE TITLE
Dataset at ftp:/podaac-ftp.jpl.nasa.gov/allData/coastal_alt

### ISSUE THREAD

**Issue by nickrsan** [climate-mirror/datasets#190]
ftp:/podaac-ftp.jpl.nasa.gov/allData/coastal_alt.

Suggested in a large email containing many urls

**Comment by blueacid** [comment]
Downloading now.

**Comment by donbright** [comment]
size estimate: 3.0T using lftp du -h

**Comment by bkirkbri** [comment]
@blueacid Please update if/when completed. Thanks!
@donbright Are you downloading this one? Thanks!

**Comment by bkirkbri** [comment]
Issue #240 is a duplicate, but is open because [a mirror was started](https://github.com/climate-mirror/datasets/issues/240#issuecomment-275535452).

**Comment by donbright** [comment]
no, i am only providing estimates so people can know whether they should start a download or not.  i dont have  space to download most of this stuff.

**Comment by bkirkbri** [comment]
@blueacid @Proplex did either of you get this one?
